tsconfig.json

```diff
@@ -3,8 +3,9 @@
         "outDir": "./dist/",
         "sourceMap": true,
         "noImplicitAny": false,
-        "module": "commonjs",
-        "target": "es6",
+        "moduleResolution": "node",
+        "module": "esnext",
+        "target": "esnext",
```

Comment:
I guess this might be incompatible with browsers, esnext is most bleeding edge target. I guess we should fix this to es2019 or whatever fully compatible with chrome/firefox.